ide/db.core/src/org/netbeans/modules/db/sql/history/SQLHistoryManager.java

```diff
@@ -97,7 +101,7 @@ public int getListSize() {
         return NbPreferences.forModule(SQLHistoryPanel.class).getInt("OPT_SQL_STATEMENTS_SAVED_FOR_HISTORY", DEFAULT_SQL_STATEMENTS_SAVED_FOR_HISTORY);
     }
 
-    protected FileObject getHistoryRoot(boolean create) throws IOException {
+    private FileObject getHistoryRoot(boolean create) throws IOException {
```

Comment:
`FileUtil.createFolder` [accepts names with forward slashes](https://bits.netbeans.org/dev/javadoc/org-openide-filesystems/org/openide/filesystems/FileUtil.html#createFolder-org.openide.filesystems.FileObject-java.lang.String-) so we're good with that `"Databases/SQLHISTORY"` string.

Reply:
If I remember correctly there is exactly one platform, that does not use "/" as separator, that is windows and I know, that windows also accepts forward slashes. I have some hope that in the future people learn from past mistakes and don't try to reinvent the wheel and either use URIs or at least use these as base.